test/test_files/ddl/ddl.test

```diff
@@ -63,3 +63,323 @@ Rel table group: likes has been created.
 -STATEMENT Call table_info('likes_person_organisation') RETURN *;
 ---- 1
 1|since|INT64
+
+-CASE CreateNodeTableCommitNormalExecution
+-STATEMENT CREATE NODE TABLE EXAM_PAPER(STUDENT_ID INT64, MARK DOUBLE, PRIMARY KEY(STUDENT_ID))
+---- ok
+
+-CASE CreateRelTableCommitNormalExecution
+-STATEMENT CREATE REL TABLE likes(FROM person TO organisation, RATING INT64, MANY_ONE)
+---- ok
+
+-CASE DropNodeTableCommitNormalExecution
+-STATEMENT DROP TABLE movies
+---- ok
+
+-CASE DropRelTableCommitNormalExecution
+-STATEMENT DROP TABLE knows
```

Comment:
Can you add a match statement after drop to verify that the table knows has been dropped.